### PR TITLE
add schemathesis to actions

### PR DESCRIPTION
**Pull request by lina-is-here**
This runs fuzzy API tests in GitHub actions.

The current results is
```
2022-08-01T09:14:47.9621732Z [36mapi-tests_1  |[0m =================================== SUMMARY ====================================
2022-08-01T09:14:47.9621865Z [36mapi-tests_1  |[0m 
2022-08-01T09:14:47.9622035Z [36mapi-tests_1  |[0m Performed checks:
2022-08-01T09:14:47.9622306Z [36mapi-tests_1  |[0m     not_a_server_error                    332 / 964 passed          FAILED 
```

### CODE REVIEW

File: backend/docker/start_fuzz_testing.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# wait for the backend to start up
+# TODO: implement something smarter
+sleep 60
```
Comment (Stranger6667) [inline backend/docker/start_fuzz_testing.sh:5]: You could do something like [this](https://github.com/schemathesis/schemathesis/blob/master/.github/workflows/example-manual-build.yml#L62):

```
          timeout 5
          bash -c
          'until printf "" 2>>/dev/null >>/dev/tcp/$0/$1; do sleep 0.2; done'
          localhost 8080
```

Comment (lina-is-here) [reply backend/docker/start_fuzz_testing.sh:5]: Thanks, I know and I've seen it too :) I created the PR before the end of my day so others are aware I'm working on it.

Comment (rsnyman) [reply backend/docker/start_fuzz_testing.sh]: @lina-is-here yeah, we do something similar in `scripts/ibutsu-pod.sh` although it is external.

**Comment by rsnyman** [comment]
This looks great to me. Shall we commit this, and then start cleaning up to fix the tests?

**Comment by lina-is-here** [comment]
@rsnyman great! Yes, that's the plan :) I'd like to work on the fixes too

**Pull request merged**